web/src/js/__tests__/components/FlowTable/FlowColumnsSpec.js

```diff
@@ -1,101 +1,101 @@
 import React from 'react'
 import renderer from 'react-test-renderer'
 import * as Columns from '../../../components/FlowTable/FlowColumns'
-import { TFlow } from '../../ducks/tutils'
+import _tflow from '../../ducks/_tflow'
```

Comment:
This is a really good point! I would like to integrate all the test mock object into `tutils.js`